core/trino-main/src/main/java/io/trino/metadata/DisabledSystemSecurityMetadata.java

```diff
@@ -222,6 +225,24 @@ public void setEntityOwner(Session session, EntityKindAndName entityKindAndName,
         throw notSupportedException(entityKindAndName.name().get(0));
     }
 
+    @Override
+    public Set<SchemaAuthorization> getSchemasAuthorizationInfo(Session session, QualifiedObjectPrefix prefix)
+    {
+        return Set.of();
```

Comment:
why not to throw? It sounds more defensive than that.

Reply:
Because other methods in this class don't throw